infra/terraform/dev/deploy/s3.tf

```diff
@@ -0,0 +1,9 @@
+resource "aws_s3_bucket" "elb_logs" {
+  bucket = "wikijump-elb-logs-${var.environment}"
+  acl    = "log-delivery-write"
+}
+
+resource "aws_s3_bucket" "wikijump_assets" {
+  bucket = "wikijump-assets-${var.environment}"
```

Comment:
What are "assets" here? Code and files?

Reply:
Yeah, actual code and images and stuff in comparison to things like logs and other AWS byproducts.